src/utils/segmentInitializer.ts

```diff
@@ -0,0 +1,47 @@
+import { Logger } from './logger';
+import Analytics from 'analytics-node';
+
+export namespace SegmentInitializer {
```

Comment:
Also there should be a way to flush the queue on deactivation.

Reply:
+1

Reply:
okay so should we call TelemetryEventQueue.dispose() , inside  deactivate() {}. 
Correct 
@arajkumar ?